### PR TITLE
Fixes #63 - documentation about grid system + styles

### LINKED ISSUE #63: add grid documentation
Depends on #15 
The Branch of 15 was merged into this one to create a proper documentation.

### PR DESCRIPTION
**Pull request by madeleine-neumann**
This PR changes the wording and css classes in the explanations which are not used anymore. It also deletes explanations which where used in the intro of the grid system. 
R? @kotzendekrabbe

### CODE REVIEW

**Comment by zoepage** [comment]
@madeleine-neumann could you resolve the merge conflicts first please? thanks!

**Comment by madeleine-neumann** [comment]
Done @zoepage

File: docs/index.html [hunk 1]
```diff
@@ -628,34 +628,16 @@ <h4 class="headline-3">Module: Typographie</h4>
 			</div>
 		</div>
 	</section>
-  
-	<section class="grid">
```
Comment (zoepage) [inline docs/index.html]: Why did you remove grid? It's still used right?

File: docs/index.html [hunk 2]
```diff
@@ -737,25 +719,25 @@ <h4 class="headline-3">Module: Colors</h4>
...
-					The <code>grid-wrapper</code> class provides the white box to hold the module as well as the padding and margin.
-				</p>
+			<div class="grid-row">
+				<div class="grid-cell x3 hint-text">
+					<h4 class="headline-3">Module: Grid - Introduction</h4>
+					<p>
+						The grid is the basic module to provide boxes for all of your modules. It is a global module, which means it is not scoped and can (also should) be used everywhere on the site.
+					</p>
+					<p>
+						The <code>grid</code> class provides a margin around the box and a hidden overflow.
```
Comment (zoepage) [inline docs/index.html]: the grid should be separated and above to make sure people see the difference, because theoretically you could build a module without grid-row. But not a grid-row / grid-col without grid.

File: docs/index.html [hunk 3]
```diff
@@ -948,7 +922,7 @@ <h4 class="headline-3">Module: 1:1:1</h4>
 				<h4 class="headline-3">Module: Grid - [2 : [1,1]] / [[1,1] : 2]</h4>
 				<p>
 					We can also wrap grid-cells together, so they will flow with the col, but not with the row what their usual behaviour is. <br />
-					The <code>element-wrapper</code> class can help us here. 
+					The <code>grid-nested</code> class can help us here. BUT - the nested elements only work if the Screen has a min-width of 665px.
```
Comment (zoepage) [inline docs/index.html]: Here I'm not sure if having a fixed value makes sense as it could change. Maybe refer to the CSS var name and add a reference like https://github.com/webcompat/design/blob/master/docs/css/variables.css#L17?

**Review by zoepage — CHANGES_REQUESTED** [review]

File: docs/index.html (hunk 3 above)
Comment (kotzendekrabbe) [reply docs/index.html]: no fix value and it's not like it's not working, it's only defined at the moment from 665 on

File: docs/index.html [hunk 4]
```diff
@@ -778,7 +760,7 @@ <h4 class="headline-3">Module: Grid - Introduction</h4>
 			<div class="grid-cell x3 hint-text">
 				<h4 class="headline-3">Module: Grid - full width</h4>
 				<p>
-					The <code>element</code> class indicates an grid-cell within the grid-box. The grid-system can be divided in 3 grid-cells (width of one grid-cell = x) that can also be combined.
+					The <code>grid-cell</code> class indicates an grid-cell within the grid-box. The grid-system can be divided in 3 grid-cells (width of one grid-cell = x) that can also be combined.
```
Comment (kotzendekrabbe) [inline docs/index.html]: i would say ..." class indicates a box within the grid"

and add an example for the cells "x1" "x2" "x3"

File: docs/index.html [hunk 5]
```diff
@@ -737,25 +719,25 @@ <h4 class="headline-3">Module: Colors</h4>
...
+					</p>
+				</div>
 			</div>
-		</div>
-	</section>
+		</section>
 	<section class="grid">
 		<article class="grid-row">
-			👈 This is the box to rule all modules. 👇 
+			👈 This is the box to rule all modules. 👇
```
Comment (kotzendekrabbe) [inline docs/index.html]: can you add an div around the text with an grid-cell please - thank you :)

File: docs/index.html (hunk 1 above)
Comment (madeleine-neumann) [reply docs/index.html]: The whole description was duplicated and introduced at line 740 as well. Maybe we should move the grid explanations to the top of the documentation?

File: docs/index.html (hunk 4 above)
Comment (madeleine-neumann) [reply docs/index.html]: We already have the explanations for them below :)

File: docs/index.html (hunk 1 above)
Comment (madeleine-neumann) [reply docs/index.html]: Grid explanations moved to the top

**Comment by madeleine-neumann** [comment]
@zoepage R?

**Comment by zoepage** [comment]
@madeleine-neumann merge conflicts

**Comment by kotzendekrabbe** [comment]
@madeleine-neumann can you resolve the merge conflicts please? thank you :)

**Comment by madeleine-neumann** [comment]
done @kotzendekrabbe
